ResearchKit/ActiveTasks/ORKTowerOfHanoiTowerView.m

```diff
@@ -66,7 +66,8 @@ - (instancetype)initWithFrame:(CGRect)frame maximumNumberOfDisks:(NSUInteger)max
 
 - (void)updateConstraints {
     if (_currentConstraints != nil) {
-        [NSLayoutConstraint deactivateConstraints:_currentConstraints];
+        [self removeConstraints:_currentConstraints];
```

Comment:
Thanks for opening the radar.
I like the idea of using a for loop, but it still crashes:

<img width="506" alt="screen shot 2015-08-06 at 4 26 12 pm" src="https://cloud.githubusercontent.com/assets/11466704/9125702/0e7be588-3c58-11e5-8dc9-6f33a010ff96.png">